### PR TITLE
Adds initial pokemon component

### PR DESCRIPTION
**Pull request by bartdorsey**
I'm adding the initial pokemon component

### CODE REVIEW

**Comment by JibronHarris** [comment]
Not enough code, BACK TO WORK

File: client/components/Pokemon.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function Pokemon() {
+  return (
+    <div id="pokemon">
```
Comment (JibronHarris) [inline client/components/Pokemon.js:3]: what is this id????

File: client/components/Pokemon.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default function Pokemon() {
```
Comment (bartdorsey) [inline client/components/Pokemon.js:1]: I thought we decided that all our components would use arrow functions

File: client/components/Pokemon.js (hunk 1 above)
Comment (bartdorsey) [reply client/components/Pokemon.js:3]: I think this should be a className....

File: client/components/Pokemon.js (hunk 2 above)
Comment (bartdorsey) [reply client/components/Pokemon.js:1]: I'm going to change this.

**Review by JibronHarris — APPROVED** [review]
This is good

**Comment by bartdorsey** [comment]
> 

Yeah but we'll add the rest when the backend is done.

**Pull request merged**